src/org/spdx/licensexml/LicenseXmlHelper.java

```diff
@@ -117,6 +117,12 @@
 		EXAMPLE_UNPROCESSED_TAGS.add(LICENSEXML_ELEMENT_BULLET);
 	}
 	
+	static HashSet<String> PHRASING_ELEMENTS = new HashSet<String>();
+	static {
+		PHRASING_ELEMENTS.add(LICENSEXML_ELEMENT_LIST);
```

Comment:
`PHRASING_ELEMENTS` [still][1] needs to be renamed to `FLOW_CONTENT_ELEMENTS (or similar)`.

[1]: https://github.com/spdx/tools/pull/122#commitcomment-25945319